go/host/p2p/metrics_util.go

```diff
@@ -2,17 +2,17 @@ package p2p
 
 import gethmetrics "github.com/ethereum/go-ethereum/metrics"
 
-// perStringGaugeMap is a map that returns metrics per string
+// perHostMetrics is a map that returns metrics per string
 // errors per host or events per type are common usages
-type perStringGaugeMap struct {
+type perHostMetrics struct {
 	gaugeMap map[string]gethmetrics.Gauge
 	instName string
```

Comment:
let's call this "instrument" or whatever the terminology used in geth is.
Is it "instrument"?

Reply:
They call it metric types. https://geth.ethereum.org/docs/interface/metrics

Should we just leave it at that ? Maybe a better name will surface once it has more usage ? Or do you prefer`perHostInstruments` ?